System/Vector.h

```diff
@@ -190,7 +190,7 @@ namespace RTE {
 		float GetMagnitude() const { return std::sqrt(std::pow(m_X, 2.0F) + std::pow(m_Y, 2.0F)); }
 
 		/// <summary>
-		/// Sets the magnitude of this Vector and keeps its angle intact.
+		/// Sets the magnitude of this Vector. A negative magnitude will invert the Vector's direction.
```

Comment:
I wonder if we want this. Maybe we should newMag = std::min(0, newMag) so it stays >= 0? Let's discuss this in discord, cause I don't really know the answer.